gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Sat Feb 24 13:39:00 EST 2018
+#Thu May 31 03:56:27 EDT 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.0-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.0-all.zip
```

Comment:
So we really don't need -all to compile this, do we? If not, this change should be reverted.

Reply:
This just downloads binary and docs files, I think it's fine downloading a few more bytes.